VideoLocker/src/main/java/org/edx/mobile/view/CourseCombinedInfoFragment.java

```diff
@@ -345,23 +336,26 @@ private void populateFacePile(){
 
     }
 
-    private View generateAnnouncementView(AnnouncementsModel model){
-
-        View convertView = inflater.inflate(R.layout.row_announcement_list, null);
-
-        TextView date = (TextView) convertView.findViewById(R.id.announcement_date);
-        TextView content = (TextView) convertView.findViewById(R.id.announcement_content);
+    private void populateAnnouncements(List<AnnouncementsModel> announcementsList) {
+        if(announcementsList !=null && announcementsList.size()>0) {
+            hideEmptyAnnouncementMessage();
 
-        date.setText(model.getDate());
+            final String divider = "<div style='height:2px; width:100%; background-color: #E2E3E5'></div>";
 
-        Spanned text = Html.fromHtml(model.content);
+            StringBuffer buff = new StringBuffer();
 
-        Spanned interceptedLinks = OutboundUrlSpan.interceptAllLinks(text);
-        content.setText(interceptedLinks);
-        content.setMovementMethod(LinkMovementMethod.getInstance());
+            // add meta viewport
+            buff.append("<meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">");
```

Comment:
@aleffert  This is the meta tag that I added. Kindly let me know if needs any change?